src/ci/docker/host-x86_64/x86_64-gnu-llvm-15/Dockerfile

```diff
@@ -0,0 +1,67 @@
+FROM ubuntu:22.10
```

Comment:
Note: this is newer because 22.04 (LTS) doesn't have llvm-15. We should probably plan to keep rolling this forward until the next LTS.